osmnx/downloader.py

```diff
@@ -595,7 +595,7 @@ def nominatim_request(params, request_type="search", pause=1, error_pause=60):
         sc = response.status_code
 
         # log the response size and domain
-        size_kb = len(response.content) / 1000.0
+        size_kb = len(response.content) / 1024
```

Comment:
Dividing by 1024 puts the size in units of kibibytes rather than kilobytes. I'm going to push a commit to keep it in kilobytes.

Reply:
See: https://en.wikipedia.org/wiki/Byte#Multiple-byte_units

Reply:
Resolved by cb41881

Reply:
Right, sorry. I am using kibibytes, but keep calling them kilobytes. Thank you for pointing that out.